### DISCUSSION TITLE
Hidden Questions

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#180] [Q&A]
I was looking at the hidden questions today, and assume this is a work in progress still? From what I can see you can create the hidden questions, and you can export them in the main export, but you are not able to enter any information into the relevant question via the admin panel? This then requires access to the DB to enter any information. 
Ideal would be to view all the questions when editing the attendee, and then edit and save from there. 
I'm looking forward to being able to utilise this great feature of hidden questions.

**Comment by daveearley** [comment]
Hidden questions were intended as a way to hide questions without permanently deleting them. A better name for them would have been "archived questions".

The ability to edit answers is definitely some useful missing functionality, so I'll add that to the roadmap. Thanks for the question!

**Comment by justynpride** [comment]
Thanks for clarifying, and for adding the feature to the road map. I can't wait for to be able to edit the answers.
